docs/api/en/loaders/FileLoader.html

```diff
@@ -18,8 +18,8 @@ <h1>[name]</h1>
 
 		<h2>Example</h2>
 		<p>
-			[example:webgl_loader_msgpack WebGL / loader / msgpack]<br />
-			[example:webgl_morphtargets_human WebGL / morphtargets / human]<br />
+			[example:webgl_loader_obj2_options WebGL / loader / obj2 / options]<br />
+			[example:webgl2_materials_texture3d WebGL2 / materials / texture3d]<br />
```

Comment:
I think we can drop this examples list since `FileLoader` is internally used in most other loaders.

Reply:
done